openshift/aws-resource-exporter.yaml

```diff
@@ -88,6 +88,24 @@ objects:
       port: 9115
       targetPort: 9115
     type: ClusterIP
+- apiVersion: v1
+  kind: ConfigMap
```

Comment:
you will need to mount this configmap as a volume into the deployment

Reply:
and make sure that `AWS_RESOURCE_EXPORTER_CONFIG_FILE` is set properly in the deployment